cluster-health-check/README.md

```diff
@@ -0,0 +1,29 @@
+# Cluster Health Check
+
+The script needs to be downloaded and run on one of the following servers.
+
+* A server or workstation with kubectl access to the local cluster.
+* Directly on one of the local cluster nodes using the `root` user or using `sudo`.
+* As a k8s deployment on the cluster.
+
+## How to use
+
+* Download the script and save as: `cluster-health.sh`
+* Make sure the script is executable: `chmod +x cluster-health.sh`
+* Run the script: `./cluster-health.sh`
+
+The script will create a .tar.gz log collection in /tmp by default, all flags are optional.
+
+## Flags
```

Comment:
IMHO using long form flags is much easier than remembering these short forms. Anyways, it's up to you.